examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,132 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example reproduces figures from Lalor et al's mTRF toolbox in
+matlab [1]_. We will show how the :class:`mne.decoding.ReceptiveField` class
+can perform a similar function along with :mod:`sklearn`. We will fit a
+linear encoding model using the continuously-varying speech envelope to
+predict activity of a 128 channel EEG system.
+
+References
+----------
+.. [1] Crosse, M. J., Di Liberto, G. M., Bednar, A. & Lalor, E. C. (2016).
+       The Multivariate Temporal Response Function (mTRF) Toolbox:
+       A MATLAB Toolbox for Relating Neural Signals to Continuous Stimuli.
+       Frontiers in Human Neuroscience 10, 604. doi:10.3389/fnhum.2016.00604
+"""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import mne
+from mne.decoding import ReceptiveField
+
+from sklearn.linear_model import Ridge
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import scale
+from scipy.io import loadmat
+from scipy.stats import pearsonr
+
+
+###############################################################################
+# Load the data from the publication
+path = mne.datasets.mtrf.data_path()
+n_decim = 2  # To speed up computation
+data = loadmat(path + '/speech_data.mat')
+raw = data['EEG'].T
+speech = data['envelope'].T
+sfreq = float(data['Fs'].squeeze())
+sfreq /= n_decim
+speech = mne.filter.resample(speech, 1, n_decim, npad='auto')
+raw = mne.filter.resample(raw, 1, n_decim, npad='auto')
```

Comment:
I think making them kwargs is a good way to support both. Many users will probably just want to use the down parameter.